README.md

```diff
@@ -54,6 +54,32 @@ genhtml -o report/ --show-details --highlight --ignore-errors source --legend lc
 grcov ~/Documents/FD/mozilla-central/build -t coveralls -s ~/Documents/FD/mozilla-central --token YOUR_COVERALLS_TOKEN > coveralls.json
 ```
 
+### GRCOV with Travis
+
+Here is an example of .travis.yml file
+```YAML
+sudo: false
+language: rust
+
+before_install:
+  - curl -L https://github.com/mozilla/grcov/releases/download/v0.4.1/grcov-linux-x86_64.tar.bz2 | tar jxf -
+
+matrix:
+  include:
+    - os: linux
+      rust: nightly
+
+script:
+    - export CARGO_INCREMENTAL=0
+    - export RUSTFLAGS="-Zprofile -Ccodegen-units=1"
```

Comment:
We could suggest disabling inlining too (-Cinline-threshold=0), and maybe forcing the linking of dead code (-Clink-dead-code) and disabling overflow checks (-Coverflow-checks=off).

We are not doing these things ourselves, maybe we should :)